CCPA/Data Deletion Request Handling.md

```diff
@@ -0,0 +1,328 @@
+
+While this specification provides the technical means for compliance in cases where CCPA applies, implementations may exist to support deletion requests outside CCPA governance. 
+
+
+### Relevant Documents
+
+[Limited Service Provider Agreement](https://www.iabprivacy.com/lspa-2019-12.pdf) (12-2-19 version)
+
+
+## How it Works
+
+Any ad tech company (vendor) supporting the US Privacy framework, exposes a JavaScript resource for each service they offer. The vendor hosts a URL that provides this resource and also serves as a unique identifier for the resource. The URL is made public to a maintained LSPA signatories list and to non-signatories based on the vendor's documented practices.
+
+Publishers working with any services provided by these vendors consume the exposed JavaScript and embed it on their properties where consumers can request deletion of their personal data. If data deletion is requested, all javascript signals on the publisher property are sent to the associated vendors. Vendors then respond by deleting the relevant personal data and signaling any affected vendors downstream.
+
+
+### Multiple Services
+
+For vendors that provide multiple services, a URL is provided for each service. A publisher may use any of those services but not necessarily all of them. When a consumer requests data deletion, only the scripts provided on the page will trigger the delete request back to the vendor. Vendors can then delete the personal data, in accordance with their practices, for each service provided on the publisher property where the request originated.
+
+
+### Deletion
+
+The technical solution detailed in this specification provides the means to signal consumer requests for data deletion. Companies supporting the US Privacy Framework (i.e., service providers) will respond to the signals by deleting the consumer's relevant personal data to the extent required by CCPA. The process for deletion depends on the company's technology and operational practices in place. _How_ a vendor deletes a consumer's personal data is out of scope for this specification.
+
+
+### Non-web Environments
+
+If the consumer initiates a data deletion request from a non-web environment, the request can be propagated through the same javascript resource described above by directing the user to a publisher webpage to complete the deletion request. The request can be associated to the device from where it originated using defined parameters.
+
+
+## Deletion Signaling
+
+Deletion signaling uses two commands: `registerDeletion` and `performDeletion`. The first command, `registerDeletion`, is executed when the publisher loads script on a given page or property. This informs the vendor that the provided script has been loaded. The second command, `performDeletion`, is executed if a consumer requests data deletion and informs the vendor that a consumer has requested that their private data be deleted.  
```

Comment:
```suggestion
Deletion signaling uses two commands: `registerDeletion` and `performDeletion`. The first command, `registerDeletion`, is executed when the publisher loads a vendor script on a given page or property. This registers the vendor script to receive the signal sent by the `performDeletion` call. The second command, `performDeletion`, is executed if a consumer requests data deletion and signals the vendor that a consumer has requested that their personal data be deleted.  
```

Not sure about the first sentence change but I am fairly sure that the call to registerDeletion does not signal the vendor in any way. Or at least, does not _need_ to signal anything to the vendor.

Reply:
@lon-pilot-mpf thanks for the edits! Proposing some further improvements for readability:

"is executed when the publisher loads a vendor script on a given page or property"
->
"is executed by the vendor script when the publisher loads it on a given page or property"

"The second command, `performDeletion`, is executed"
->
"The second command, `performDeletion`, is executed by the publisher"